frontend/src/routes/(frontend)/about/index.tsx

```diff
@@ -1,66 +1,76 @@
 import { component$ } from '@builder.io/qwik'
 import { DocumentHead, loader$ } from '@builder.io/qwik-city'
+import { getDatabase } from 'wildebeest/backend/src/database'
 import { getDomain } from 'wildebeest/backend/src/utils/getDomain'
+import { handleRequestGet as settingsHandleRequestGet } from 'wildebeest/functions/api/wb/settings/server/server'
+import { handleRequestGet as rulesHandleRequestGet } from 'wildebeest/functions/api/v1/instance/rules'
 import { Accordion } from '~/components/Accordion/Accordion'
-import { AccountCard } from '~/components/AccountCard/AccountCard'
+// import { AccountCard } from '~/components/AccountCard/AccountCard'
 import { HtmlContent } from '~/components/HtmlContent/HtmlContent'
-import { george } from '~/dummyData/accounts'
+import { ServerSettingsData } from '~/routes/(admin)/settings/server-settings/layout'
 import { Account } from '~/types'
 import { getDocumentHead } from '~/utils/getDocumentHead'
 import { instanceLoader } from '../layout'
+import { getAdmins } from 'wildebeest/functions/api/wb/settings/server/admins'
+import { emailSymbol } from 'wildebeest/backend/src/activitypub/actors'
+import { loadLocalMastodonAccount } from 'wildebeest/backend/src/mastodon/account'
+import { AccountCard } from '~/components/AccountCard/AccountCard'
+import { getNotFoundHtml } from '~/utils/getNotFoundHtml/getNotFoundHtml'
 
 type AboutInfo = {
 	image: string
 	domain: string
-	contact: {
-		account: Account
-		email: string
-	}
-	rules: { id: string; text: string }[]
+	admin: { account: Account | null; email: string }
+	rules: { id: number; text: string }[]
 	extended_description: {
-		updated_at: string
 		content: string
 	}
 }
 
-export const aboutInfoLoader = loader$<Promise<AboutInfo>>(async ({ resolveValue, request, redirect }) => {
-	// TODO: properly implement loader and remove redirect
-	throw redirect(302, '/')
+export const aboutInfoLoader = loader$<Promise<AboutInfo>>(async ({ resolveValue, request, platform, html }) => {
+	throw html(404, getNotFoundHtml())
```

Comment:
Note: this PR doesn't make the about page accessible (I'll do that in my next PR)